README.md

```diff
@@ -4,45 +4,98 @@
-
-- GitHub reusable workflows: *.github/workflows/*
-- Landing Page for the [MORYX-CodeCoverage-Website](https://d2g620u22bjjnb.cloudfront.net/): *src/CodeCoverage-Pages/*
-- Action to sync Landing Page to AWS Server (in progress)
 
-## Adding the workflow to MOYRX-Repositories
+This repo contains reusable GitHub workflows, used by the MORYX team maintain 
+their CI pipelines.
 
-The current standard workflow in MORYX-Repositories includes:
+## Example
 
-- Buildung the solution and package it
-- Unit-Testing with coverage information collection
-- Integration-Testing with coverage information collection
-- Generating a coverage report via ReportGenerator
-- Syncing coverage html-pages to AWS-S3-Bucket
-- Documentation
-- Publishing built packages to Myget or Nuget
+```yml
+jobs:
+  Variables:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set Release variables 
+        id: set-variables
+        run: echo ''
+    outputs:
+      dotnet_sdk_version: 6
+      PACKAGE_TARGET: <nuget-package-feed>
+      PACKAGE_TARGET_V3: <nuget-package-feed/v3>
+      NPM_PACKAGE_SOURCE: <npm-package-source>
 
-To add this workflow to your MORYX-Repository, follow these steps:
+  Build:
+    needs: [Variables]
+    uses: moryx-industry/tools/.github/workflows/build.yml@v1
+    with:
+      dotnet_sdk_version: ${{ needs.Variables.outputs.dotnet_sdk_version }}
+      NPM_PACKAGE_SOURCE: ${{ needs.Variables.outputs.NPM_PACKAGE_SOURCE }}
+    secrets:
+      npm_auth_token: ${{ secrets.MYGET_TOKEN }}
+      myget_auth_token: ${{ secrets.MYGET_TOKEN }}
+      myget_user: ${{ secrets.MYGET_USER }}
+      myget_pass: ${{ secrets.MYGET_PASS }}
 
- 1. Add the nuget-package "coverlet.collector" to your test-projects and also add a package-reference with version to *Directory.build.targets*.
- 2. Make sure, you have the *.build*-directory with all its content.
- 3. Add the needed secrets to your repository: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, MYGET_TOKEN, NUGET_TOKEN.
- 4. Add the *build-and-test-tool.yml* as *build-and-test.yml* to your repository to *.github/workflows/* in your repository.
+  Tests:
+    needs: [Variables, Build]
+    uses: moryx-industry/tools/.github/workflows/run-unit-tests.yml@v1
+    with:
+      dotnet_sdk_version: ${{ needs.Variables.outputs.dotnet_sdk_version }}
+      npm_package_source: ${{ needs.Variables.outputs.NPM_PACKAGE_SOURCE }}
+    secrets:
+      npm_auth_token: ${{ secrets.MYGET_TOKEN }}
+      myget_auth_token: ${{ secrets.MYGET_TOKEN }}
+      myget_user: ${{ secrets.MYGET_USER }}
+      myget_pass: ${{ secrets.MYGET_PASS }}
 
-For reference, the workflow is currently in use in the follwing repositories: [MORYX-Framework](https://github.com/PHOENIXCONTACT/MORYX-Framework) and [MORYX-Factory](https://github.com/PHOENIXCONTACT/MORYX-Factory).
+  Licensing:
+    needs: [Tests]
+    uses: moryx-industry/tools/.github/workflows/licensing.yml@v1
 
-### Adding the repository coverage results to the MORYX-CodeCoverage-Website
```

Comment:
I would not remove this part, or was your intention to only have that in the pxc repo? 🤔